app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import classname from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  classIconRule,
+  iconRule,
+  isStoryDontEstimated,
```

Comment:
IMO, naming this variable `isStoryEstimated` would read better and then you should update the logic wherever necessary since this changes the meaning of the variable.
If you don't like changing the logic for this variable, I would suggest changing its name to `isStoryNotEstimated`.

Reply:
Nice, I change the variable's name!

Reply:
Thanks, for the tip!